src/lib/routes/admin-api/public-signup.ts

```diff
@@ -245,15 +249,16 @@ export class PublicSignupController extends Controller {
         req: IAuthRequest<void, void, PublicSignupTokenCreateSchema>,
         res: Response<PublicSignupTokenSchema>,
     ): Promise<void> {
+        const userName = extractUsername(req);
         const token =
             await this.publicSignupTokenService.createNewPublicSignupToken(
                 req.body,
-                req.user.name,
+                userName,
```

Comment:
Wondering about the capitalization here: is this a user's name or username? The function name reads like it a username, but the variable reads as if it's a user's name. Maybe change the variable to `username` if that's what it is? The same goes for lines 290 and 292

Reply:
yes misspelled it